### PR TITLE
Form filler initial manifest properties

### PR DESCRIPTION
**Pull request by OasisNeupane**
Added the basic structure to the manifest file.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "manifest_version": 1,
+    "name": "Manifest file structure",
```
Comment (Bikash-Gurung) [inline manifest.json:3]: Please add the name of our application "FormFiller"

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "manifest_version": 1,
```
Comment (Bikash-Gurung) [inline manifest.json:2]: I think It should be 3. Did you try to run the application(extension) in your computer? Did it run successfully?

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Manifest file structure",
+    "description": "basic sturcutre of the manifest.json file",
```
Comment (Bikash-Gurung) [inline manifest.json:4]: Write a little description about our FormFiller application, what it does.

File: manifest.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "0.0.1",
+    "author": "oasisneupane@gmail.com",
```
Comment (Bikash-Gurung) [inline manifest.json:6]: Since, Abhinav is also working on this project, you can remove author field.

Comment (Bikash-Gurung) [reply manifest.json:6]: Or you can add abhinav's email as well

File: manifest.json (hunk 1 above)
Comment (OasisNeupane) [reply manifest.json:3]: done

File: manifest.json (hunk 2 above)
Comment (OasisNeupane) [reply manifest.json:2]: done

File: manifest.json (hunk 3 above)
Comment (OasisNeupane) [reply manifest.json:4]: done

File: manifest.json (hunk 4 above)
Comment (OasisNeupane) [reply manifest.json:6]: done

**Review by Bikash-Gurung — APPROVED** [review]

**Pull request merged**